pkg/controller/atlasproject/atlasproject_controller.go

```diff
@@ -49,10 +49,16 @@ type AtlasProjectReconciler struct {
 	OperatorPod client.ObjectKey
 }
 
+// Dev note: duplicate the permissions in both sections below to generate both Role and ClusterRoles
```

Comment:
[q] am I right that ClusterRole is for the cluster-wide operator, while Role is for the namespaced one?

Reply:
yes, absolutely